core/build.gradle

```diff
@@ -37,31 +38,32 @@ coveralls {
 
 buildscript {
     ext.support_library_version = '28.0.0'
-    ext.butterknife_version = '8.8.1'
-    ext.dagger_version = '2.13'
+    ext.butterknife_version = '10.1.0'
```

Comment:
I believe we can remove the `butterknife` version and depenency since it's not used anymore.